Jenkinsfile

```diff
@@ -0,0 +1,27 @@
+pipeline {
+  agent {
+    node {
+      label 'bare-metal'
+    }
+
+  }
+  stages {
+    stage('Build') {
+      steps {
+        sh 'docker run --rm -v `pwd`:/go/src/github.com/intel/pmem-csi pmem-csi-ci:go-alpine bash -c "make"'
+      }
+    }
+    stage('Test') {
+      steps {
+        sh 'docker run --rm -v `pwd`:/go/src/github.com/intel/pmem-csi pmem-csi-ci:go-alpine bash -c "make test"'
```

Comment:
How do we control which version of Go is used? This has to be configured in the pmem-csi repo, because it may depend on the branch and has to be changed together with the code.

Reply:
This gets resolved by doing everything inside the build image specified by our own Dockerfile.